style.css

```diff
@@ -0,0 +1,54 @@
+ul{
+    float: right;
+    padding: 20px 30px;
+}
+li {
+    display: inline-block;
+    margin: 10px 15px;    
+}
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 60%;
+    padding: 70px 90px;
```

Comment:
i will suggest you reduce the font-size of the google image and flex to center the image